File: src/algorithms/count-elements.js
/**
 * Given an integer array nums, returns the number of elements that have both
 * a strictly smaller and a strictly greater element appear in nums.
 * 
 * @param {number[]} nums - An array of numbers
 * @returns {number} - Returns a count of elements that have both strictly 
 *     greater and strictly smaller elements
 */
function countElements (nums) {
  let count = 0;
  nums.sort((a, b) => {
    return a - b;
  });
  let min = nums[0];
  let max = nums[nums.length - 1];
  for (let i = 1; i < nums.length - 1; i++) {
    if (nums[i] > min && nums[i] < max) {
      count++;
    }
  }
  return count;
}

File: src/algorithms/count-elements.py
def count_elements(nums):
    """Given an integer array nums, return the number of elements that have 
    both a strictly smaller and a strictly greater element appear in nums.

    Args:
        nums (list[int]): A list of integers.

    Returns:
        int: Returns a count of elements that have both a strictly smaller and
        a strictly greater element appear in nums.
    """
    count = 0
    nums = sorted(nums)
    min_num = nums[0]
    max_num = nums[len(nums) - 1]
    for i in range(1, len(nums) - 1):
        if min_num < nums[i] and max_num > nums[i]:
            count += 1
    return count
